src/chrome/content/toolbar_button.xul

```diff
@@ -43,6 +43,8 @@
       <menupopup id="https-everywhere-context" onpopupshowing="show_applicable_list(this)">
         <!-- entries will be written here by ApplicableList.populate_menu() -->
         <menuseparator class="hide-on-disable"/>
+        <menuitem label="&https-everywhere.prefs.reset_defaults;"
```

Comment:
You spotted my corner cutting. The problem is, the string for "Reset to defaults" is in the .dtd locale file (accessible only from XUL), but the menu subsection with the ruleset togglers is built dynamically in Javascript. I considered porting over the "Reset to defaults" string into the .properties locale file so it can be accessed from Javascript, but it didn't seem worth the time.